.github/workflows/build-test-publish-images.yml

```
@@ -87,35 +87,35 @@ jobs:
             pr_num="${GITHUB_REF_NAME##*/}"
             BASE_TAG_PREFIX="docker-${pr_num}-"
             NOTEBOOKS_TAG_PREFIX="docker-notebooks-${pr_num}-"
-            RAFT_ANN_BENCH_TAG_PREFIX="docker-raft-ann-bench-${pr_num}-"
-            RAFT_ANN_BENCH_DATASETS_TAG_PREFIX="docker-raft-ann-bench-datasets-${pr_num}-"
-            RAFT_ANN_BENCH_CPU_TAG_PREFIX="docker-raft-ann-bench-cpu-${pr_num}-"
+            CUVS_BENCH_TAG_PREFIX="docker-cuvs-bench-${pr_num}-"
+            CUVS_BENCH_DATASETS_TAG_PREFIX="docker-cuvs-bench-datasets-${pr_num}-"
+            CUVS_BENCH_CPU_TAG_PREFIX="docker-cuvs-bench-cpu-${pr_num}-"
           fi
           echo "BASE_TAG_PREFIX=${BASE_TAG_PREFIX}" | tee -a ${GITHUB_OUTPUT}
           echo "NOTEBOOKS_TAG_PREFIX=${NOTEBOOKS_TAG_PREFIX}" | tee -a ${GITHUB_OUTPUT}
-          echo "RAFT_ANN_BENCH_TAG_PREFIX=${RAFT_ANN_BENCH_TAG_PREFIX}" | tee -a ${GITHUB_OUTPUT}
-          echo "RAFT_ANN_BENCH_DATASETS_TAG_PREFIX=${RAFT_ANN_BENCH_DATASETS_TAG_PREFIX}" | tee -a ${GITHUB_OUTPUT}
-          echo "RAFT_ANN_BENCH_CPU_TAG_PREFIX=${RAFT_ANN_BENCH_CPU_TAG_PREFIX}" | tee -a ${GITHUB_OUTPUT}
+          echo "CUVS_BENCH_TAG_PREFIX=${CUVS_BENCH_TAG_PREFIX}" | tee -a ${GITHUB_OUTPUT}
+          echo "CUVS_BENCH_DATASETS_TAG_PREFIX=${CUVS_BENCH_DATASETS_TAG_PREFIX}" | tee -a ${GITHUB_OUTPUT}
+          echo "CUVS_BENCH_CPU_TAG_PREFIX=${CUVS_BENCH_CPU_TAG_PREFIX}" | tee -a ${GITHUB_OUTPUT}
       - name: Compute image repo
         id: compute-image-repo
         run: |
           base_repo="base"
           notebooks_repo="notebooks"
-          raft_ann_bench_repo="raft-ann-bench"
-          raft_ann_bench_datasets_repo="raft-ann-bench-datasets"
-          raft_ann_bench_cpu_repo="raft-ann-bench-cpu"
+          CUVS_bench_repo="cuvs-bench"
```

Let's match case, either all upper or all lower. This applies from here on down.
```suggestion
          cuvs_bench_repo="cuvs-bench"
```